dev-docs/howtoguides/release_a_new_version.md

```diff
@@ -39,174 +49,215 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
+    still unsure about the behaviour of a feature or when we fix a bug that removes the code
+    that was added. We must verify each changelog entry that is added to be sure of their
+    presence in the product.
+
+
+### 2. Set up `review` and `next` branches
+
+```bash
+# Make sure you are up to date
+git switch main
+git pull
+
+# Tag the release candidate
+git tag $version-rc # e.g., 42-rc
+git push origin $version-rc
+
+# Create the next branch
+git switch -c next-v$((version+1))
+dch --newversion $((version+1))~1~devel
+git add debian/changelog
+git commit -m "open next development version"
+git push origin next-v$((version+1))
+
+# Create the review branch
+git switch main
+git switch -c review-v$version
+git push origin review-v$version
+```
+
+Now configure two things:
+1. Change GitHub's "default" branch to `next-v$((version+1))`
+    - This should automatically switch any open PRs against `main` to target `next-v$((version+1))`.
+2. Change the daily PPA build recipe to use `next-v$((version+1))`
+
+And finally, create a draft PR from `main` to `review-v$version` on GitHub.
+
+### 3. Release to our staging PPA
+
+1. Clone the repository into a clean directory and `git checkout $version`.
+    * *WARNING* Build the package in a clean environment. The reason is that the package
+    will contain everything that is present in the folder. If you are storing credentials or
+    other sensible development information in your folder, they will be uploaded too when we send
+    the package to the ppa. A clean environment is the safest way to perform this.
+
+2. Edit the changelog
+  * List yourself as the author of this release.
+  * Edit the version number to look like: `$version~rc1` (`<version>~rc<release-candidate-number>`)
+  * Edit the Ubuntu release name. Start with the ubuntu/devel release.
+  * `git add debian/changelog && git commit -m "throwaway"` - Do **not** push this commit!
+
+3. Run `build-package`
+  * This script will generate all the package artifacts in the parent directory as `../out`.
+
+4. Run `sbuild-it ../out/<package_name>.dsc`
+  * If this succeeds move on. If this fails, debug and fix before continuing.
+
+5. Repeat steps 2 through 4 for all supported Ubuntu Releases
+  * The version for series other than devel should be in the form `<version>~<ubuntu-release-number>~rc<release-candidate-number>`
+  This means you must add the release number in the changelog. For example, suppose
+  the devel version is `1.1~rc1`. If you want to build for jammy now, change it to
+  `1.1~22.04~rc1`.
+
+6. For each release, dput to the staging PPA:
+  * `dput ppa:ua-client/staging ../out/<package_name>_source.changes`
+  * After each `dput` wait for the "Accepted" email from Launchpad before moving on.
+
+### 4. Release to Ubuntu (devel and SRU)
 
 1. Prepare SRU Launchpad bugs.
+    * We do this even before a successful merge into ubuntu/devel because the context added to these bugs is useful for the Server Team reviewer.
```

Comment:
```suggestion
    * We do this even before a successful merge into `ubuntu/devel` because the context added to these bugs is useful for the Server Team reviewer.
```